packages/js-drive/lib/abci/handlers/initChainHandlerFactory.js

```diff
@@ -109,6 +113,12 @@ function initChainHandlerFactory(
 
     const validatorSetUpdate = createValidatorSetUpdate(validatorSet);
 
+    proposalBlockExecutionContext.setCoreChainLockedHeight(initialCoreChainLockedHeight);
+
+    const coreChainLockUpdate = await createCoreChainLockUpdate(0, consensusLogger);
+
+    proposalBlockExecutionContext.reset();
```

Comment:
It looks strange? Why should we do that? Maybe we need to decompose createCoreChainLockUpdate?